dev/devicelab/bin/tasks/ios_platform_view_xcuitests.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
`native_ui_platform_view_tests_ios` to better match `native_ui_tests_macos`?

You'll need to add an entry to `.ci.yaml` to get this to run anywhere:
```
  - name: Mac_ios native_ui_platform_view_tests_ios
    bringup: true
    recipe: devicelab/devicelab_drone
    presubmit: false
    timeout: 60
    properties:
      tags: >
        ["devicelab", "ios", "mac"]
      task_name: native_ui_platform_view_tests_ios
    scheduler: luci
```